python/setup.py

```diff
@@ -81,9 +81,12 @@ def get_llvm_package_info():
         system_suffix = f"macos-{arch}"
     elif system == "Linux":
         # TODO: arm64
```

Comment:
Should you delete "TODO"?

Reply:
just pushed commit to delete that, should have done that earlier